numpy/f2py/tests/test_compile_function.py

```diff
@@ -81,12 +82,15 @@ def test_f2py_init_compile(extra_args):
             return_check = import_module(modname)
             calc_result = return_check.foo()
             assert_equal(calc_result, 15)
+            # Removal from sys.modules, is not as such necessary. Even with
+            # removal, the module (dict) stays alive.
+            del sys.modules[modname]
 
 
 def test_f2py_init_compile_failure():
     # verify an appropriate integer status value returned by
     # f2py.compile() when invalid Fortran is provided
-    ret_val = numpy.f2py.compile(b"invalid")
+    ret_val = numpy.f2py.compile(b"invalid", verbose=False)
```

Comment:
curious why you changed this?

Reply:
Oh, those are those spammy comilation prints you say if you run the tests with `-- -s`... and they seem pretty unimportant to pirnt out by default.

Reply:
This assumes anyone looking for what went wrong in the tests will know to add `verbose=True` in whatever test interests them

Reply:
maybe add a comment somewhere?

Reply:
Maybe, it would probably be nicestest to store it in a variable. In a sense that is what pytest does. I think part of it was that pytest leaks had issues with it (which are fixed). So I just remove it for now, can be added easily at any time.